### DISCUSSION TITLE
[Question]: Overload Coral

### DISCUSSION THREAD

**Discussion by ghost** [blakeblackshear/frigate#13433] [Ask A Question]
### What is your question?

Hello,
I have a dual EdgeTPU with 4 cameras, detection FPS 9.
I'm using an other model so my interfence speed is around 185ms.
What happens if there are more movement like in a windy day? Can it overload my Coral? If yes, what happens?
Thanks for your help.

**Comment by NickM-27** [comment]
The coral already is overloaded, that inference speed is too show to maintain 9 fps even on one camera

**Reply by ghost** [reply]
I see, and how Frigate handle this?
Alerts seems to appear correctly.

**Reply by NickM-27** [reply]
frigate will skip the frames, you will see skipped frames in the camera metrics, this is not good and you want to avoid this
